include/openssl/ocsp.h

```diff
@@ -33,11 +33,19 @@ extern "C" {
 // validation path for the signer certificate unless the OCSP_NOCHAIN flag is
 // set.
 #define OCSP_NOCHAIN 0x8
+// OCSP_NOVERIFY is for |OCSP_basic_verify|. This is a no-op flag in AWS-LC.
+// When setting this flag in OpenSSL, the |OCSP_BASICRESP|'s signature will
+// still be verified, but setting this flag skips verifying the signer's
+// certificate.
+#define OCSP_NOVERIFY 0x10
```

Comment:
In the past when we don't support something we've set it to 0.
```suggestion
#define OCSP_NOVERIFY 0
```